tests/integration_tests/functional/test_rate_limiter.py

```diff
@@ -13,7 +13,7 @@
 IPERF_TRANSMIT_TIME = 4
 
 # Use a fixed-size TCP window so we get constant flow
-IPERF_TCP_WINDOW = '1000K'
+IPERF_TCP_WINDOW = '256K'
```

Comment:
A number of issues logged ([iperf#1199](https://github.com/esnet/iperf/issues/1199), [iperf#757](https://github.com/esnet/iperf/issues/757)) that contain discussions around the bug that existed in iperf 3.1 that did not check that the buffer/window configuration was compatible with the OS configuration.